SWLOR.Game.Server/Data/Migrations/2020-10-22.2.sql

```diff
@@ -0,0 +1,63 @@
+﻿UPDATE Perk
+Set IsActive = 0
+WHERE ID = 1;
+
+UPDATE Perk
+Set IsActive = 0
+WHERE ID = 10;
+
+UPDATE Perk
+Set IsActive = 0
+WHERE ID = 79;
+
+UPDATE Perk
+Set IsActive = 0
+WHERE ID = 98;
+
+UPDATE Perk
+Set IsActive = 0
+WHERE ID = 101;
+
+UPDATE Perk
+Set IsActive = 1
+WHERE ID = 175;
+
+Update PerkFeat
```

Comment:
The IDs on the following PerkFeats are not right. Use a combination of FeatId AND PerkLevelUnlocked and do a retest.